src/main/java/com/jalasoft/sdfc/pages/common/BasicForm.java

```diff
@@ -24,6 +26,20 @@
             "following-sibling::input[@type='checkbox']";
     private static final String FOOTER_BUTTON = "//div[@class = 'inlineFooter']/" +
             "descendant::span[text() = '%s']/parent::button";
+    // It allows to identify if a field in the form is required, use this after trying to save the item
+    private static final String MARKED_AS_REQUIRED_FIELD = "//div[contains(@class, 'has-error')]/" +
+            "descendant::span[starts-with(text(),'%s')]";
+
+    @FindBy(css = ".errorsList > li")
```

Comment:
are you sure this locator will retrieve a single value?, what happens if there are 2 error messages?

Reply:
yeah, it may be possible to have more than 1 error message, so I updated the locator